packages/vue/src/components/atoms/SfBreadcrumbs/SfBreadcrumbs.vue

```diff
@@ -10,17 +10,22 @@
         <template v-if="last !== i">
           <!-- @slot Custom markup for previous pages (binds `breadcrumb` object) -->
           <slot name="link" v-bind="{ breadcrumb, go }">
-            <a class="sf-breadcrumbs__breadcrumb" @click="go(breadcrumb)">{{
-              breadcrumb.text
-            }}</a>
+            <SfLink
+              :link="breadcrumb.route.link"
+              class="sf-breadcrumbs__breadcrumb"
+              @click="go(breadcrumb)"
+              >{{ breadcrumb.text }}
+            </SfLink>
           </slot>
         </template>
         <template v-else>
           <!-- @slot Custom markup for current page (binds `breadcrumb` object) -->
           <slot name="current" v-bind="{ breadcrumb }">
-            <span
+            <a
```

Comment:
@justyna-13 maybe we can use here SfLink here and remove v-focus from this component :)